shared/settings/account/confirm-delete.tsx

```diff
@@ -32,24 +33,39 @@ const getPrompt = (props: Props) =>
     </Kb.Box2>
   )
 
+const getDescription = (props: Props) => {
+  let description = ''
```

Comment:
```
[
...(props.lastEmail ? [`Since you'll have deleted all email addresses, you won't get email notifications from Keybase anymore.`] : []),
...(props.searchable ? [`Your friends will no longer be able to find you by this ${
      props.type === 'email' ? 'email address' : 'number'
    }.`] : [])
].join(' ')
```